### DISCUSSION TITLE
[ livewire v3 ]  wire:click , wire:* not working within bootstrap datatable

### DISCUSSION THREAD

**Discussion by Ola-Yusuf** [livewire/livewire#7169] [1. Help]
I just upgraded from Livewire v2 to v3.

I have actions triggered with the wire:click within the bootstrap datatable

all the wire:click and other wire events within the bootstrap datatable is not working 

No error message is shown

I tried to update the jQuery used but the wire:* is still not working

**Comment by Seiliaz** [comment]
it's probabely from the endpoints.
check if there's no wrong endpoints in the ServiceProvider. 
after that, if it wasn't still ok; remove livewire. then remove composer.lock and vendor directory. 
after that, write the command "composer install" and then reinstall livewire. it will fix
